sdk/storage/azure-storage-queue/tests/_shared/asynctestcase.py

```diff
@@ -8,6 +8,7 @@
 import asyncio
 import functools
 
+from azure_devtools.scenario_tests.patches import mock_in_unit_test
```

Comment:
We now also need this update added to azure-storage-file-share, which has the same `_shared` module :)

Reply:
It was different when I created this PR 😉